api/src/main/java/gov/va/vro/api/resources/ClaimMetricsResource.java

```diff
@@ -149,4 +155,41 @@ ResponseEntity<List<ExamOrderInfoResponse>> allExamOrderInfo(
       @RequestParam(name = "size", required = false, defaultValue = "10")
           @Min(value = 1, message = "invalid size")
           Integer size);
+
+  @Operation(
+      summary =
+          "Retrieves health evidence for a specific claimSubmissionId, claim version,"
+              + " veteran ICN, and diagnostic code.",
+      description =
+          "This endpoint does the same thing v2/health-data-assessment used to do. "
+              + " It does not store/write anything to the database.")
+  @PostMapping(value = "/health-evidence")
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "200", description = "Successful"),
+        @ApiResponse(
+            responseCode = "401",
+            description = "Unauthorized",
+            content = @Content(schema = @Schema(hidden = true))),
+        @ApiResponse(
+            responseCode = "404",
+            description = "Not found",
+            content = @Content(schema = @Schema(hidden = true))),
+        @ApiResponse(
+            responseCode = "500",
+            description = "Internal service error",
+            content = @Content(schema = @Schema(hidden = true)))
+      })
+  @ResponseStatus(HttpStatus.OK)
+  @Timed(value = "health-evidence")
+  @Tag(name = "Claim Metrics")
+  @ResponseBody
+  ResponseEntity<HealthDataAssessment> healthEvidence(
```

Comment:
Also i'm not sure if the HDA object here has everything we want. you may need FullHealthDataAssessmentResponse instead to expose the sufficientEvidence flag.